clock: 20
bebop101:
  initial_pose:
    x: 1.0
    y: 0
    z: 1
bebop102:
  initial_pose:
    x: -1.0
    y: 0
    z: 1
observe_field:
  mesh_acc: [105,105,5, 5,11] #x,y,z, yaw, pitch (should be 0 in linspace for compress)
  limit: [[-1.5, 1.5],[-1.5,1.5],[0.0,0.5],[0.1,0.8], [-3.14,3.14]] #[x[min,max], theta[min,max], z[min,max]] 
drone_field:
  mesh_acc: [105, 105] #x (should contain 0 in linspace for compress)
  limit: [[-1.5, 1.5], [-1.5, 1.5]] #[x[min,max]] 
rviz_field:
  mesh_acc: [105, 105] #x (should contain 0 in linspace for compress)
  limit: [[-1.5, 1.5], [-1.5, 1.5]] #[x[min,max]] 
theta_field:
  mesh_acc: [5, 11] #x (should contain 0 in linspace for compress)
  limit: [[0.1,0.8], [-3.14,3.14]] #[x[min,max]] 


sigma: 0.1